ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml.cs

```diff
@@ -18,44 +19,81 @@ public class UserPage : BaseLayout
 
     public bool CommentsEnabled;
 
-    public bool IsProfileUserHearted;
+    public List<SlotEntity>? HeartedSlots;
```

Comment:
Is this meant to be in alphabetical order? If it is, then you should have the same types grouped together then the groups should be sorted alphabetically.